docs/api/autogluon.task.rst

```diff
@@ -111,35 +112,25 @@ Prediction tasks built into AutoGluon such that a single call to `fit()` can pro
         :methods:
 
 
-Additional Tabular Prediction APIs
+Additional Tabular APIs
 ----------------------------------
 
-.. automodule:: autogluon.tabular.tabular_prediction
-
-:hidden:`TabularPredictor`
-~~~~~~~~~~~~~~~~~~~~~~~~~~
 .. automodule:: autogluon.tabular
 
-.. autoclass:: TabularPredictor
-   :members:
-   :no-inherited-members:
-
-    .. rubric:: Methods
-
-    .. autoautosummary:: TabularPredictor
-        :methods:
-
-    .. rubric:: Attributes
-
-    .. autoautosummary:: TabularPredictor
-        :attributes:
-
 :hidden:`TabularDataset`
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. autoclass:: TabularDataset
```

Comment:
Currently this documents every pd.DataFrame method, which we don't want. Unsure how to hide all of those methods. I tried `:no-methods:` but that removed all documentation of TabularDataset, when we still want `__init__`.

Reply:
maybe can try something like: `:methods: X Y` where X, Y are the only methods to list in documentation

Reply:
Will address in follow-up